### ISSUE TITLE
How do I properly add newly generated docs to mruby.github.io?

### ISSUE THREAD

**Issue by Mav7** [mruby/mruby.github.io#45]
The last bit of documentation I wrote was approved of and merged by @matz into [mruby](https://github.com/mruby/mruby/pull/3183). Now I want to add the documentation to [mruby.org](http://mruby.org/). Usually what would happen is that I'd generate the doc using [mruby-yard](https://github.com/sagmor/yard-mruby). In this case, the only thing that changed was that  I added docs to string.h. So I'd then take the generated doc in this case  a file called mruby_2Fstring.h.html. Then I'd just copy and paste it to the proper directory in [mruby.github.io](http://mruby.github.io/) then I'd do a PR. Of course, if it's merged it would then go live. Is that still okay? Because I remember before @zzak said that the size [being too big](https://github.com/mruby/mruby.github.io/pull/39). I want to make sure I do this correctly this time!

**Comment by zzak** [comment]
It looks like @sagmor wrote [a script to do this](https://github.com/mruby/mruby.github.io/blob/master/gen/api), but I'm not sure.

**Comment by Mav7** [comment]
@zzak That script generates the docs from the mruby repo. I'm not sure what to do next about the PR. I used the scipt ``./gen/api`` then when I run git status I get a long list like this:
```
...
modified:   docs/api/String.html
modified:   docs/api/Struct.html
modified:   docs/api/Symbol.html
...
```
It still seems like it will make for a huge PR.

**Comment by zzak** [comment]
@Mav7 Right, yeah my guess is because they just haven't been generated in a while

**Comment by Mav7** [comment]
@zzak So how would I go about doing PR then? It's still seems to be to much.

**Comment by zzak** [comment]
@Mav7 I mean, I think it's fine to send a PR to update the site -- but in the long run it's better to have this done automatically.

**Comment by pusewicz** [comment]
@matz This is now resolved by https://github.com/mruby/mruby.github.io/pull/262 as we build the docs on deploy.
